numaprom/udf/filter.py

```diff
@@ -12,7 +12,7 @@
 
 @catch_exception
 @msg_forward
-def metric_filter(_: str, datum: Datum) -> Optional[Messages]:
+def metric_filter(_: List[str], datum: Datum) -> Optional[Messages]:
```

Comment:
```suggestion
def metric_filter(_: list[str], datum: Datum) -> Optional[Messages]:
```
Since this is Python 3.10 only